pages/partials/attachFilePanel.ejs

```diff
@@ -51,52 +51,67 @@
     </div>
     <% } else { /* assessment_instance is open and authorized_edit is true */ %>
     <!-- UPLOAD FILE -->
-    <button class="btn btn-xs btn-secondary" type="button" data-toggle="collapse" data-target="#attachFileCollapse" aria-expanded="false" aria-controls="attachFileCollapse">
-      Attach a file <i class="far fa-caret-square-down"></i>
-    </button>
-    <div class="collapse" id="attachFileCollapse">
-      <form class="attach-file-form" name="attach-file-form" method="POST" enctype="multipart/form-data" class="mb-3">
-        <p class="small mt-3">
-          Attached files will be saved here for your reference.
-        </p>
-        <div class="form-group">
-          <div class="custom-file">
-            <input type="file" name="file" class="custom-file-input" id="attachFileInput">
-            <label class="custom-file-label" for="attachFileInput">Choose file</label>
-            <small id="emailHelp" class="form-text text-muted">Max file size: <%= config.fileUploadMaxBytesFormatted %></small>
+    <div>
```

Comment:
Recommended to review with whitespace changes hidden. While I was here, I changed from a `<br>` to wrapping each section in a `<div>` to avoid content jumping around when the panels are expanding and collapsing.